setup.py

```diff
@@ -93,9 +102,12 @@ def run(self):
         _datafiles()
         _develop.run(self)
 
+get_patches()
```

Comment:
The return value isn't being used but the global `data_files` is being modified.